include/CL/sycl/accessor.hpp

```diff
@@ -589,14 +590,34 @@ accessMode == access::mode::discard_read_write) && dimensions == 1) */
 
 
   /* Available only when: accessMode == access::mode::atomic && dimensions == 0*/
-  //operator atomic<dataT, access::address_space::global_space> () const;
+  template<int D = dimensions,
+           access::mode M = accessmode,
+           typename = std::enable_if_t<M == access::mode::atomic && D == 0>>
+  HIPSYCL_UNIVERSAL_TARGET
+  operator atomic<dataT, access::address_space::global_space> () const
+  {
+      return atomic<dataT, access::address_space::global_space> { global_ptr<dataT>(_ptr) };
+  }
 
   /* Available only when: accessMode == access::mode::atomic && dimensions > 0*/
-  //atomic<dataT, access::address_space::global_space> operator[](
-  //    id<dimensions> index) const;
+  template<int D = dimensions,
+           access::mode M = accessmode,
+           typename = std::enable_if_t<(D > 0) && (M == access::mode::atomic)>>
+  HIPSYCL_UNIVERSAL_TARGET
+  atomic<dataT, access::address_space::global_space> operator[](
+      id<dimensions> index) const
+  {
+      return atomic<dataT, access::address_space::global_space> { global_ptr<dataT>(_ptr + detail::linear_id<dimensions>::get(index, _buffer_range)) };
```

Comment:
Final nitpick: Can you introduce a linebreak here? This line is twice as long as the other lines in this file ;) Also: We use an indentation of 2 spaces.

Reply:
Done, I think.